src/common/core/cli/run.py

```
@@ -0,0 +1,79 @@
+
+def _migrate() -> None:
+    _wait_for_db()
+    databases: list[str] = getattr(settings, "FLAGSMITH_MIGRATE_DATABASES", ["default"])
+    for database in databases:
+        django_execute_from_command_line(
+            ["flagsmith", "migrate", "--database", database]
+        )
+    django_execute_from_command_line(["flagsmith", "createcachetable"])
+
+
+def migrate(argv: list[str], *, prog: str) -> None:
```

Ruff (0.15.20)

[warning] Unused function argument: `prog`

(ARG001)